includes/User.php

```diff
@@ -1668,8 +1669,8 @@ public function isBlocked( $bFromSlave = true, $shouldLogBlockInStats = true ) {
 	 *
 	 * @return Block|null
 	 */
-	public function getBlock( $bFromSlave = true, $shouldLogBlockInStats = true ){
-		$this->getBlockedStatus( $bFromSlave, $shouldLogBlockInStats );
+	public function getBlock( $bFromSlave = true, $shouldLogBlockInStats = true, $global = true ){
```

Comment:
:speech_balloon: — for backwards compatibility, assume we are dealing with a global block.